lib/atoms/img/example.html

```diff
@@ -0,0 +1 @@
+<img ng-src=" " />
```

Comment:
We need to add an angular parameter for ng-src (e.g. ng-src="{{url}}"). Also class attribute must be added, in order to customize the example from patterns configuration.